### ISSUE TITLE
Add number of easy, medium and hard issues solved to API response

### ISSUE THREAD

**Issue by hs2361** [kunaljain0212/git-init-FOSS-API#1]
Currently, we are only assigning the score for each student (10 points for easy, 20 for medium and 30 for hard). However, we would like to also add the number of easy, medium and hard issues that students solve to the leaderboard API response. After this, the API response should look something like this:
```
{
...
  "username": {
    "name": "name",
    "rollNumber": "2020XXX-YYY",
    "score": S,
    "easy": E,
    "medium": M,
    "hard": H,
  },
...
}
```

The score calculation logic is here
https://github.com/kunaljain0212/git-init-FOSS-API/blob/6f488909e3b509e4cbcff5458b46b40f40602db2/src/updaterDB.js#L566-L590

This issue is fairly simple to solve given that the checks for easy, medium and hard issues are already present.

**Comment by hs2361** [comment]
@AmanPachori Sure, go ahead. Assigning this to you.

**Comment by AmanPachori** [comment]
I have doubt that 
 I have to add
    "easy": E,
    "medium": M,
    "hard": H,  this for every student

**Comment by hs2361** [comment]
@AmanPachori Yes, those will be added for every student. E, M and H represent the number of easy, medium and hard problems that the student has solved. Take a look at the [score.json file](https://github.com/kunaljain0212/git-init-FOSS-API/blob/main/src/score.json) for reference.
